### PR TITLE
Functional Tests : Migration TS (BO - 04)

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Functional Tests : Migration TS (BO - 04)
| Type?             | improvement
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Relative to #30263
| Related PRs       | N/A
| How to test?      | Nightly is :green_circle: : https://github.com/Progi1984/ga.tests.ui.pr/actions/runs/3831019732
| Possible impacts? | N/A

### CODE REVIEW

**Review by boubkerbribri — DISMISSED** [review]

File: tests/UI/campaigns/functional/BO/04_customers/03_outstanding/03_viewInvoice.ts [hunk 1]
```diff
@@ -3,48 +3,26 @@ import files from '@utils/files';
...
-let filePath;
-
-// New order by customer data
-const orderByCustomerData = {
-  customer: DefaultCustomer,
-  productId: 1,
-  productQuantity: 1,
-  paymentMethod: PaymentMethods.wirePayment.moduleName,
-};
+const baseContext = 'functional_BO_customers_outstanding_viewInvoice';
```
Comment (nesrineabdmouleh) [inline tests/UI/campaigns/functional/BO/04_customers/03_outstanding/03_viewInvoice.ts]:
```suggestion
const baseContext: string = 'functional_BO_customers_outstanding_viewInvoice';
```

**Review by boubkerbribri — DISMISSED** [review]

**Review by nesrineabdmouleh — COMMENTED** [review]
A small feedback :)

File: tests/UI/campaigns/functional/BO/04_customers/03_outstanding/03_viewInvoice.ts (hunk 1 above)
Comment (Progi1984) [reply tests/UI/campaigns/functional/BO/04_customers/03_outstanding/03_viewInvoice.ts]: Fixed in 3884fdf

**Review by nesrineabdmouleh — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]

**Review by boubkerbribri — APPROVED** [review]

**Comment by boubkerbribri** [comment]
Thanks @Progi1984

**Pull request merged**